src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,45 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String color;
```

Comment:
Machine already has `color` - why do we write this field in child classes as well?